plugins/obs-ffmpeg/obs-ffmpeg-mux.c

```diff
@@ -565,6 +568,23 @@ static void save_replay_proc(void *data, calldata_t *cd)
 	UNUSED_PARAMETER(cd);
 }
 
+static void save_replay_proc_secs(void *data, calldata_t *cd)
+{
+	int64_t secs;
+
+	// maybe should save full RB if no valid value is found
```

Comment:
I think this was discussed in Discord, but did this ever get decided?

Reply:
Yes, it was discussed in Discord on November 7th, 2019.

Seemed that the general idea was to send a message to the log and not save anything, but given that there is no way to warn the user that nothing was saved in that moment (only after reading the log), I'd prefer it saved the whole buffer instead, as to avoid loss of data.

Do we discuss it further here or in Discord, or is there another place where it's more "official" to discuss?

Matt | WizardCM suggested it could be an RFC, but I'm not sure if he was referring to this or the "user feedback system" that Fenrir was talking about in Discord.